src/components/panel/panel.spec.js

```diff
@@ -499,8 +557,51 @@ describe('$mdPanel', function() {
 
       openPanel(config);
       expect(PANEL_EL).toHaveClass(FULLSCREEN_CLASS);
+    });
 
+    it('should default backdrop to false', function() {
+      openPanel(DEFAULT_CONFIG);
+      expect(BACKDROP_CLASS).not.toExist();
     });
+
+    it('should show backdrop when hasBackdrop=true', function() {
+      var config = { template: DEFAULT_TEMPLATE, hasBackdrop: true };
+
+      openPanel(config);
+      expect(BACKDROP_CLASS).toExist();
+
+      closePanel();
+      expect(BACKDROP_CLASS).not.toExist();
```

Comment:
Should you add a real class to the backdrop, which would also help debugging in the future? Perhaps, `md-panel-backdrop`? You're checking for `'.md-opaque-enter'`, which is an animation class.